cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/GoalBasedOperationRunnable.java

```diff
@@ -38,7 +38,7 @@
   protected final boolean _dryRun;
   protected final boolean _stopOngoingExecution;
   protected final boolean _skipHardGoalCheck;
-  protected final Pattern _excludedTopics;
+  protected final Pattern _excludedTopicsPattern;
```

Comment:
Nit: (Applies to similar changes) Such global name changes makes the PR harder to review as it touches many files that are irrelevant to the current PR. 

Can we please move such global name changes to their own trivial PRs?